mex/editor/models.yaml

```diff
@@ -0,0 +1,46 @@
+AccessPlatform:
+  title: title
+  preview:
+    - alternativeTitle
+    - unitInCharge
+Activity:
+  title: title
+  preview:
```

Comment:
no, it's just how the configuration for how to display each model type in the frontend for example when you do a search or have a link field.

see https://github.com/robert-koch-institut/mex-editor/pull/117/files#diff-241def25481c545bfed47d68895cbbeb903b70358b8c52b189f341491ca9d1e2R15

because not all models have a "title" and "description" field we could use, each model type will look a little different.

Reply:
got it.